docs/_docs/Instructors/04-Recommendations-High-Quality-materials.md

```diff
@@ -1,64 +1,62 @@
 
-## Always use real word motivations 
-Always start by _"why this is worth learning"_, by showing a practical case in which a concept can be applied. It is much harder for students to reason about "two random variables, X and Y" than it is to reason about "let's take a bunch of kids, where we know their ages (X) and heights (Y)". By anchoring the solution to a real problem you will help the students to form memories and intuitions, as well as being much more motivating. 
+## Use real word motivations 
+Always start with _"why this is worth learning"_. Show a practical case in which a concept can be applied. It is much harder for students to reason about "two random variables, X and Y" than it is to reason about "let's take a bunch of kids, where we know their ages (X) and heights (Y)". By anchoring the solution to a real problem you will help the students to form memories and intuitions, as well as being much more motivating. 
 
 ## How much theory? 
-There is a delicate trade-off between making sure the students know why they are using something, and feeding them theory for theory's sake. Be practical, and think whether a piece of theoretical knowledge has helped you significantly to use a particular concept. 
+There is a delicate trade-off between making sure the students know why they are using something, and feeding them theory for theory's sake. Be practical, and think whether a piece of theoretical knowledge has helped you to use a particular concept. 
 
-As an example, having an intuition of Gradient Descent helps understand why a Logistic Regression needs to be normalized, and is therefore worth including. On the other hand being able to derive the Cross Entropy error function is mostly fun-for-nerds, and arguably does not particularly enhance our ability to use it in practice. 
+As an example, having an intuition of Gradient Descent helps understand why a Logistic Regression needs to be normalized, and is therefore worth including. On the other hand being able to derive the Cross Entropy error function is mostly fun-for-nerds, and does not particularly enhance our ability to use it in practice. 
 
-Ensure that everything we teach is there to make them be able to use the concepts, while avoiding over-correcting and only teaching "how to do stuff". Never teach something just because you had to suffer through it yourself, or because you happen to find it cool, as remember that you have years of experience, and a completely different base. Whenver in doubt, ask the head of Curriculum (in our case João Ascensão). 
+Ensure that everything we teach is there to make the students able to use the concepts, while avoiding teaching only "how to do stuff". Never teach something just because you had to suffer through it yourself, or because you happen to find it cool, as remember that you have years of experience, and a completely different base. Whenever in doubt, ask the Curriculum lead. 
```

Comment:
```suggestion
Ensure that everything we teach is there to make the students able to use the concepts, while avoiding teaching only "how to do stuff". Never teach something just because you had to suffer through it yourself, or because you happen to find it cool, as remember that you have years of experience, and a completely different base. Whenever in doubt, ask the Teaching lead. 
```